go/base/static/js/src/apps/dialogue/models.js

```diff
@@ -182,6 +183,39 @@
       }).defaults(GroupStateModel.__super__.defaults.call(this));
     }
   });
+  
+  var HttpJsonStateModel = DialogueStateModel.extend({
+    storableOnContact: false,
+
+    relations: [{
+      type: Backbone.HasOne,
+      key: 'url',
+      includeInJSON: 'url'
+    }, {
+      type: Backbone.HasOne,
+      key: 'method',
+      includeInJSON: 'method'
+    }, {
+      type: Backbone.HasOne,
+      key: 'entry_endpoint',
+      relatedModel: DialogueEndpointModel
+    }, {
+      type: Backbone.HasOne,
+      key: 'exit_endpoint',
+      relatedModel: DialogueEndpointModel
+    }],
+
+    defaults: function() {
+      return _({
+        text: '',
+        url: '',
+        http_method: '',
+        httpmeth: '',
+        entry_endpoint: {},
+        exit_endpoint: {}
+      }).defaults(GroupStateModel.__super__.defaults.call(this));
```

Comment:
Ah, one last comment here: I think `http_method` should be `method` to match what we have in the relations above.